north/src/npk/linux.rs

```diff
@@ -127,7 +127,7 @@ async fn install_internal(
     let start = time::Instant::now();
 
     if let Some(npk_name) = npk.file_name() {
-        info!("Loading {}", npk_name.to_string_lossy());
+        info!("Installing {}", npk_name.to_string_lossy());
```

Comment:
To be honest, "Installing" sounds like a one-time-operation after download, not something done on every boot...